ctapipe/containers.py

```diff
@@ -56,13 +57,56 @@
     "StatisticsContainer",
     "IntensityStatisticsContainer",
     "PeakTimeStatisticsContainer",
+    "SchedulingBlockContainer",
+    "ObservationBlockContainer",
+    "ObservationConfigurationContainer",
+    "ObservingMode",
+    "ObservationBlockState",
 ]
 
 
 # see https://github.com/astropy/astropy/issues/6509
 NAN_TIME = Time(0, format="mjd", scale="tai")
 
 
+class SchedulingBlockType(enum.Enum):
+    """
+    Types of Scheduling Block
+    """
+
+    OBSERVATION = 0
+    CALIBRATION = 1
+    ENGINEERING = 2
+    SIMULATION = 3
+
+
+class ObservationBlockState(enum.Enum):
+    """
+    Observation Block State, from XXX
+    """
+
+    FAILED = 0
+    COMPLETED_SUCCEDED = (1,)
+    COMPLETED_CANCELED = (2,)
+    COMPLETED_TRUNCATED = (3,)
+    ARCHIVED = 4
+
+
+class ObservingMode(enum.Enum):
+    """How a scheduling block is observed."""
+
+    WOBBLE = 0
+    ON_OFF = 1
+    GRID = 2
+    CUSTOM = 3
+
+
+class CoordinateFrameType(enum.Enum):
+    ALTAZ = 0
+    ICRS = 1
+    GALACTIC = 2  # is this needed?
```

Comment:
No, not for pointing mode, since a fixed pointing in GALACTIC is also fixed in ICRS.

Reply:
true, so I will define this separately